charts/karmada/values.yaml

```diff
@@ -828,6 +828,8 @@ schedulerEstimator:
       maxSurge: 50%
   ## @param apiServer.podDisruptionBudget
   podDisruptionBudget: *podDisruptionBudget
+  ## @param featureGate to schedulerEstimator
+  featureGates: {}
```

Comment:
If I want to use this filed, what content should I write into it?

Can you give a example as comments above `featureGates: {}`